node/src/actors/chain_manager/mod.rs

```diff
@@ -1195,10 +1196,22 @@ impl ChainManager {
                         // Store the ARS and the order of the keys
                         let ars_identities = ARSIdentities::new(ars_members);
 
+                        // Committee size should decrease if sufficient epochs have elapsed since last confirmed superblock
+                        // Committee_size = expected_size - ((current_index-last_consolidated_index)/decrease_period))
+
+                        let committee_size = calculate_committee_size(
+                            consensus_constants
+                                .superblock_signing_committee_size,
+                            consensus_constants.superblock_agreement_decreasing_period,
+                            chain_info.highest_superblock_checkpoint.checkpoint,
+                                superblock_index
+                        );
+                        log::error!("{} is the size", committee_size);
```

Comment:
You can modify this log to be a properly DEBUG log message

Reply:
This was not supposed to be part of the PR hahaha

Reply:
but I will